CHANGELOG.md

```diff
@@ -1,5 +1,86 @@
 # Change Log
 
+## [1.0.0](https://github.com/stellargraph/stellargraph/tree/v1.0.0)
+
+[Full Changelog](https://github.com/stellargraph/stellargraph/compare/v0.11.0...v1.0.0)
+
+This 1.0 release of StellarGraph is the culmination of 2 years of activate development.
+
+Jump in to this release, with the new demos and examples:
+
+- [More helpful indexing and guidance in demo READMEs](demos/)
+- [Loading from Neo4j][neo4j-1.0.0]
+- [More explanatory Node2Vec link prediction][n2v-lp-1.0.0]
+- [Unsupervised `GraphSAGE` and `HinSAGE` via `DeepGraphInfomax`][dgi-1.0.0]
+- Graph classification [with `GCNSupervisedGraphClassification`][gc-gcn-1.0.0] and [with `DeepGraphCNN`][gc-dgcnn-1.0.0]
+- [Time series prediction using spatial information, using `GraphConvolutionLSTM`][gcn-lstm-1.0.0] (experimental)
+
+[neo4j-1.0.0]: demos/basics/loading-saving-neo4j.ipynb
+[n2v-lp-1.0.0]: demos/link-prediction/random-walks/cora-lp-demo.ipynb
+[dgi-1.0.0]: demos/embeddings/deep-graph-infomax-cora.ipynb
+[gc-gcn-1.0.0]: demos/graph-classification/supervised-graph-classification.ipynb
+[gc-dgcnn-1.0.0]: demos/graph-classification/dgcnn-graph-classification.ipynb
+[gcn-lstm-1.0.0]: demos/spatio-temporal/gcn-lstm-LA.ipynb
+
+### Major features and improvements
+
+- Better demonstration notebooks and documentation to make the library more accessible to new and existing users:
+  - Notebooks are [now published in the API documentation][demos-1.0.0], for better & faster rendering and more convenient access [\#1279](https://github.com/stellargraph/stellargraph/pull/1279) [\#1433](https://github.com/stellargraph/stellargraph/pull/1433) [\#1448](https://github.com/stellargraph/stellargraph/pull/1448)
+  - The [demos indices][demos-1.0.0] and [READMEs](demos/) now contain more guidance and explanation to make it easier to find a relevant example [\#1200](https://github.com/stellargraph/stellargraph/pull/1200)
+  - Several demos have been added or rewritten: [loading data from Neo4j][neo4j-1.0.0] [\#1184](https://github.com/stellargraph/stellargraph/pull/1184), [link prediction using Node2Vec][n2v-lp-1.0.0] [\#1190](https://github.com/stellargraph/stellargraph/pull/1190), [graph classification with GCN][gc-gcn-1.0.0], [graph classification with DGCNN][gc-dgcnn-1.0.0]
+  - Notebooks now detect if they're being used with an incorrect version of the StellarGraph library, eliminating confusion about version mismatches [\#1242](https://github.com/stellargraph/stellargraph/pull/1242)
+  - Notebooks are easier to download, both individually via a button on each in the API documentation [\#1460](https://github.com/stellargraph/stellargraph/pull/1460) and in bulk via [a `curl` command]() [\#1377](https://github.com/stellargraph/stellargraph/pull/1377) [\#1459](https://github.com/stellargraph/stellargraph/pull/1459)
```

Comment:
Is the `curl` command supposed to be pointing to a link? Otherwise can be
```suggestion
  - Notebooks are easier to download, both individually via a button on each in the API documentation [\#1460](https://github.com/stellargraph/stellargraph/pull/1460) and in bulk via a `curl` command [\#1377](https://github.com/stellargraph/stellargraph/pull/1377) [\#1459](https://github.com/stellargraph/stellargraph/pull/1459)
```

Reply:
Oh, yeah, I forgot. I was going to link to https://stellargraph.readthedocs.io/en/v1.0.0/demos/index.html#download-the-demos (i.e. the 1.0.0 version of `develop`'s https://stellargraph.readthedocs.io/en/latest/demos/index.html#download-the-demos).

This reminds me: should all of our other links to demos be to https://stellargraph.readthedocs.io/en/v1.0.0/demos/... rather than the relative links?

Reply:
> This reminds me: should all of our other links to demos be to https://stellargraph.readthedocs.io/en/v1.0.0/demos/... rather than the relative links?

Yeah that's probably better, I think the experience has become a lot nicer on rtd so should just try to point there where possible. I'll update the links